dev-python/gitana/gitana-0.2.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+#PYTHON_REQ_USE="xml"
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="SQL-based Git Repository Inspector"
+HOMEPAGE="https://github.com/SOM-Research/Gitana"
+SRC_URI="https://github.com/SOM-Research/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+KEYWORDS="~amd64 ~x86"
+
+# problem with https://github.com/SOM-Research/Gitana/issues/5
+# wait for release and update version of git-python as soon as possible
+
+DEPEND="dev-lang/python:2.7[tk]
+	dev-python/git-python:0[${PYTHON_USEDEP}]
+	dev-python/mysql-connector-python:0[${PYTHON_USEDEP}]
+	dev-python/networkx:0[${PYTHON_USEDEP}]
+	dev-python/pillow:0[tk,${PYTHON_USEDEP}]
+	dev-python/PyGithub:0[${PYTHON_USEDEP}]
+	dev-python/python-bugzilla:0[${PYTHON_USEDEP}]
+	dev-python/simplejson:0[${PYTHON_USEDEP}]
+	>=dev-vcs/git-1.9.4
+	>=virtual/mysql-5.6"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/Gitana-${PV}"
+
+src_prepare() {
+	cp "${FILESDIR}/${PV}/init.py" "${S}/__init__.py" \
+		|| die "Could not copy '${FILESDIR}/${PV}/init__.py' to '${S}/__init.py'"
+	cp "${FILESDIR}/${PV}/setup.py" "${S}/setup.py" \
+		|| die "Could not copy '${FILESDIR}/${PV}/setup.py' to '${S}/setup.py'"
+	epatch "${FILESDIR}/${PV}/case_study.py.patch"
+	epatch "${FILESDIR}/${PV}/db2json_gui.py.patch"
+	epatch "${FILESDIR}/${PV}/git2db.py.patch"
+	epatch "${FILESDIR}/${PV}/git2db_gui.py.patch"
+	epatch "${FILESDIR}/${PV}/gitana.py.patch"
+	epatch "${FILESDIR}/${PV}/gitana_gui.py.patch"
+	epatch "${FILESDIR}/${PV}/gitquerier_gitpython.py.patch"
+	epatch "${FILESDIR}/${PV}/init_dbschema.py.patch"
+	epatch "${FILESDIR}/${PV}/updatedb_gui.py.patch"
+	epatch "${FILESDIR}/${PV}/updatedb.py.patch"
+	distutils-r1_python_prepare_all
+	eapply_user
+}
+
+python_install() {
+	distutils-r1_python_install
+	# install config file
+	dodir /etc/
+	local MY_SITEDIR=$(python_get_sitedir)
+	local MY_FILE="${MY_SITEDIR}/${PN}/config_db.py"
+	local MY_CONF="/etc/gitana_db.conf"
+	mv "${ED}${MY_FILE}" "${ED}${MY_CONF}" \
+		|| die "Could not move '${ED}${MY_FILE}' to '${ED}${MY_CONF}"
+	dosym "${MY_CONF}" "${MY_FILE}"
+	ewarn "Please edit ${EPREFIX}/etc/gitana_db.conf" \
+		" with settings for your MySQL server."
+	# install executable gitana_gui in /usr/bin
+	local MY_SCRIPT="${MY_SITEDIR}/${PN}/gitana_gui.sh"
+	echo "${PYTHON} ${MY_SITEDIR}/${PN}/gitana_gui.py" \
+		>> "${ED}${MY_SCRIPT}" \
+		|| die "Could not create '${ED}${MY_SCRIPT}"
```

Comment:
If I just use `dosym "${MY_SITEDIR}/${PN}/gitana_gui.py" "/usr/bin/gitana_gui"` then I get the following error message as soon as I start the program:
`~ $ gitana_gui
Traceback (most recent call last):
  File "/usr/bin/gitana_gui", line 63, in <module>
    main()
  File "/usr/bin/gitana_gui", line 60, in main
    GUI()
  File "/usr/bin/gitana_gui", line 17, in __init__
    self.initialize()
  File "/usr/bin/gitana_gui", line 22, in initialize
    img = ImageTk.PhotoImage(Image.open(self.mypath + "img/gitana_icon.png"))
  File "/usr/lib64/python2.7/site-packages/PIL/Image.py", line 2282, in open
    fp = builtins.open(filename, "rb")
IOError: [Errno 2] No such file or directory: '/usr/bin/img/gitana_icon.png'`